app/screens/AppNavigator/screens/Settings/SettingsScreen.tsx

```diff
@@ -22,6 +22,8 @@ export function SettingsScreen (): JSX.Element {
           <RowNetworkItem key={index} network={network} />
         ))
       }
+      <SectionTitle text={translate('screens/Settings', 'SUPPORT')} testID='tools_and_support_title' />
```

Comment:
Should not put support, just leave a blank header for this.

Reply:
updated